src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizardTaloudelliset.js

```diff
@@ -1,29 +1,26 @@
 import React from "react";
-import { MUUTOS_WIZARD_TEKSTIT } from "../modules/constants";
 import FormSection from "../../../../../../components/03-templates/FormSection";
 import { injectIntl } from "react-intl";
 import TaloudellisetYleisettiedot from "./Taloudelliset/TaloudellisetYleisettiedot";
 import TaloudellisetInvestoinnit from "./Taloudelliset/TaloudellisetInvestoinnit";
 import TaloudellisetTilinpaatostiedot from "./Taloudelliset/TaloudellisetTilinpaatostiedot";
 import TaloudellisetLiitteet from "./Taloudelliset/TaloudellisetLiitteet";
+import wizard from "../../../../../../i18n/definitions/wizard";
 import PropTypes from "prop-types";
 import * as R from "ramda";
 
 const MuutospyyntoWizardTaloudelliset = ({
   changeObjects,
   muutoshakemus,
   onChangesUpdate,
-  onStateUpdate
+  onStateUpdate,
+  intl: { formatMessage }
```

Comment:
There shouldn't be a need to define formatMessage. This should be fixed now or later.